src/__tests__/Calculator.test.js

```diff
@@ -0,0 +1,7 @@
+import renderer from 'react-test-renderer';
+import Calculator from '../components/Calculator';
+
+it('should render Calculator', () => {
+  const renderCalculator = renderer.create(<Calculator />).toJSON();
+  expect(renderCalculator).toMatchSnapshot();
+});
```

Comment:
As asked by the requirement, please create unit tests using  **React Testing Library** to simulate user interaction.

<img width="889" alt="image" src="https://user-images.githubusercontent.com/7243132/199620896-966d496d-6013-4608-96a7-a26e191f4eae.png">